lib/private/Lock/DBLockingProvider.php

```diff
@@ -39,21 +39,14 @@
  * Locking provider that stores the locks in the database
  */
 class DBLockingProvider extends AbstractLockingProvider {
-	private IDBConnection $connection;
-	private ITimeFactory $timeFactory;
 	private array $sharedLocks = [];
-	private bool $cacheSharedLocks;
 
 	public function __construct(
-		IDBConnection $connection,
-		ITimeFactory $timeFactory,
-		int $ttl = 3600,
-		bool $cacheSharedLocks = true
+		private IDBConnection $connection,
+		private ITimeFactory $timeFactory,
+		protected int $ttl = 3600,
```

Comment:
ttl property is from parent class, it’s ugly to redeclare it here.

Reply:
```suggestion
		int $ttl = 3600,
```
Maybe add a constructor to parent class to be able to call parent::__construct($ttl) ?